### ISSUE TITLE
question? 

### ISSUE THREAD

**Issue by 6clicks** [vishalbasnet23/woodivi#1]
is it possible to use it with DIVI theme instead for divi builder?

**Comment by vishalbasnet23** [comment]
Yes @6clicks if you have Divi Builder bundled withing the Divi theme it should work but it can not work without Divi Builder.
